UnitTest/TestComponentCSharp_Tests.cs

```diff
@@ -525,15 +525,17 @@ public void TestComposedNonBlittableStruct()
             Assert.True(val.bools.x);
             Assert.False(val.bools.y);
             Assert.True(val.bools.z);
-        }
-
+        }
+
+#if NETCOREAPP2_0
```

Comment:
is the failing test?  if so can you add a comment/TODO?

Reply:
This is not, but this test uses the ABI type directly which is not possible with IDIC anymore so I removed it from .net5